basilisk/gui/history_msg_text_ctrl.py

```
@@ -506,21 +506,32 @@ def on_show_as_html(self, event: wx.Event | None):
 			self.GetParent(), self.current_msg_content, "markdown"
 		)
 
-	def append_stream_chunk(self, text: str):
-		"""Append a chunk of text to the speech stream buffer.
+	@property
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)